code/datums/mutantraces.dm

```diff
@@ -836,6 +836,38 @@
 			mob.base_body_temp = initial(mob.base_body_temp)
 		. = ..()
 
+	emote(var/act, var/voluntary)
+		switch(act)
+			if ("whip")
+				if (mob.emote_check(voluntary, 50))
+					. = list("<B>[mob]</B> whips their tail", "<i>whips their tail</i>")
+			if ("hiss")
+				if (mob.emote_check(voluntary, 50))
+					. = list("<B>[mob]</B> hisses", "<i>hisses</i>")
+			if ("dewlap")
+				if (mob.emote_check(voluntary, 50))
+					. = list("<B>[mob]</B> extends dewlap proudly", "<i>extends dewlap proudly</i>")
```

Comment:
I think it'd sound better as "Mr Lizard extends his/her/their dewlap proudly". And/or "shamefully".

Reply:
the heck is a dewlap